cwf/gateway/configs/templates/uesim.yml

```diff
@@ -0,0 +1,9 @@
+---
+radius_address: "127.0.0.1:1812"
+radius_auth_address: "127.0.0.1:1812"
+radius_acct_address: "127.0.0.1:1813"
+radius_secret: "888888"
+amf: "0000"
+subscribers:
+        "001010000000081": {msisdn: "1234567890", apn: "wifi", rat: 6}
```

Comment:
per `getConfiguredSubscribers` many parameters like `auth_key` are required, or the client will fail.  You will have to add fake values. Do not make them optional unless you add some logic to decide what parameters are needed for each case (either UEsim or UESimHSSless)

Reply:
I plan not make any parameters mandatory. In the add_ue command, OP is mandatory. What i feel is may be that can also be removed as it can be defaulted to the same OP for multiple IMSIs. But i did not made that change yet as its an existing code.

Reply:
I will address this in the refactoring diff that i am posting as an update to this diff.